addons/l10n_tn/data/account_fiscal_position_template_data.xml

```diff
@@ -0,0 +1,128 @@
+<odoo>
+    <record id="l10n_tn_fp_template_domestic" model="account.fiscal.position.template">
+        <field name="sequence">1</field>
+        <field name="name">Domestic market</field>
+        <field name="chart_template_id" ref="l10n_tn_chart_template"/>
+        <field name="auto_apply" eval="True"/>
+        <field name="vat_required" eval="True"/>
+        <field name="country_id" ref="base.tn"/>
+    </record>
+    <record id="l10n_tn_fp_template_exo" model="account.fiscal.position.template">
+        <field name="name">Exemption</field>
+        <field name="chart_template_id" ref="l10n_tn_chart_template"/>
+        <field name="note">Exonération de TVA marché domestique</field>
+    </record>
+    <record id="l10n_tn_fp_template_export" model="account.fiscal.position.template">
+        <field name="name">Export</field>
+        <field name="chart_template_id" ref="l10n_tn_chart_template"/>
+        <field name="note">Export 0% VAT/Import</field>
+    </record>
```

Comment:
Note speaks about export/import but the name doesn't?

Shouldn't this one be auto applied too? Otherwise I don't see the point of `l10n_tn_fp_template_domestic` Maybe I'm missing something?